src/Tribe/PUE/Checker.php

```diff
@@ -635,16 +632,23 @@ public function get_network_domain() {
 		 * @return Tribe__PUE__Utility An instance of Tribe__PUE__Utility, or NULL when no updates are available.
 		 */
 		public function request_update() {
-			//For the sake of simplicity, this function just calls request_info()
-			//and transforms the result accordingly.
-			$pluginInfo = $this->request_info( array( 'pu_checking_for_updates' => '1' ) );
-			if ( $pluginInfo == null ) {
+			// For the sake of simplicity, this function just calls request_info()
+			// and transforms the result accordingly.
+			$this->plugin_info = $pluginInfo = $this->request_info( array( 'pu_checking_for_updates' => '1' ) );
+
+			if ( null === $pluginInfo ) {
 				return null;
 			}
-			//admin display for if the update check reveals that there is a new version but the API key isn't valid.
+
+			// admin display for if the update check reveals that there is a new version but the API key isn't valid.
 			if ( isset( $pluginInfo->api_invalid ) ) { //we have json_error returned let's display a message
-				$this->json_error = $pluginInfo;
-				add_action( 'admin_notices', array( &$this, 'display_json_error' ) );
+				/**
+				 * For backwards compatibility this will be kept in the code for 2 versions
+				 *
+				 * @deprecated
+				 * @todo  remove on 4.5
+				 */
+				$this->json_error = $this->plugin_info;
```

Comment:
The docblock complete with `@deprecated` tag would normally be added where the field is declared (line 39) - could we move it there?